coriolis/secrets.py

```diff
@@ -10,6 +10,7 @@
 
 
 def get_secret(ctxt, secret_ref):
+    keystone.create_trust(ctxt)
```

Comment:
NIT: this could lead to a lot of leftover trusts.
If really needed, I'd recycle this old PR I dumped I a while back:
https://github.com/cloudbase/coriolis/pull/22/files#diff-5a2302b43cadb0681e8b33764c884578R37